R/util1_OrderCpGsByLocation.R

```diff
@@ -52,7 +55,7 @@ OrderCpGsByLocation <- function(
     )
   }
   CpGs.gr <- sort(
-    CpGlocations.gr[ CpGs_char[goodCpGs_lgl] ]
+    CpGlocations.gr[CpGs_char[goodCpGs_lgl], ignore.strand = ignoreStrand]
```

Comment:
In this form, we would have an "invalid subsetting" error. Notice that the comma is inside the call to `[`. We need to make sure that the `ignore.strand` argument is being passed to the `sort()` function from `BiocGenerics::`.